airflow/providers/databricks/operators/databricks.py

```diff
@@ -160,6 +162,142 @@ def get_link(
+        webhook_notifications: j.JobWebhookNotifications | None = None,
+        timeout_seconds: int | None = None,
+        schedule: j.CronSchedule | None = None,
+        max_concurrent_runs: int | None = None,
+        git_source: j.GitSource | None = None,
+        access_control_list: list[j.AccessControlRequest] | None = None,
+        databricks_conn_id: str = "databricks_default",
+        polling_period_seconds: int = 30,
+        databricks_retry_limit: int = 3,
+        databricks_retry_delay: int = 1,
+        databricks_retry_args: dict[Any, Any] | None = None,
+        **kwargs,
+    ) -> None:
+        """Creates a new ``DatabricksJobsCreateOperator``."""
+        super().__init__(**kwargs)
+        self.json = json or {}
+        self.databricks_conn_id = databricks_conn_id
+        self.polling_period_seconds = polling_period_seconds
+        self.databricks_retry_limit = databricks_retry_limit
+        self.databricks_retry_delay = databricks_retry_delay
+        self.databricks_retry_args = databricks_retry_args
+        if name is not None:
+            self.json["name"] = name
+        if tags is not None:
+            self.json["tags"] = tags
+        if tasks is not None:
+            self.json["tasks"] = [task.as_dict() for task in tasks]
+        if job_clusters is not None:
+            self.json["job_clusters"] = [job_cluster.as_dict() for job_cluster in job_clusters]
+        if email_notifications is not None:
+            self.json["email_notifications"] = email_notifications.as_dict()
+        if webhook_notifications is not None:
+            self.json["webhook_notifications"] = webhook_notifications.as_dict()
+        if timeout_seconds is not None:
+            self.json["timeout_seconds"] = timeout_seconds
+        if schedule is not None:
+            self.json["schedule"] = schedule.as_dict()
+        if max_concurrent_runs is not None:
+            self.json["max_concurrent_runs"] = max_concurrent_runs
+        if git_source is not None:
+            self.json["git_source"] = git_source.as_dict()
+        if access_control_list is not None:
+            self.json["access_control_list"] = [acl.as_dict() for acl in access_control_list]
+
+        self.json = normalise_json_content(self.json)
+
+    @cached_property
+    def _hook(self):
+        return self._get_hook(caller="DatabricksJobsCreateOperator")
+
+    def _get_hook(self, caller: str) -> DatabricksHook:
+        return DatabricksHook(
+            self.databricks_conn_id,
+            retry_limit=self.databricks_retry_limit,
+            retry_delay=self.databricks_retry_delay,
+            retry_args=self.databricks_retry_args,
+            caller=caller,
+        )
+
+    def execute(self, context: Context) -> int:
+        if "name" not in self.json:
+            raise AirflowException("Missing required parameter: name")
+        job_id = self._hook.find_job_id_by_name(self.json["name"])
+        if job_id is None:
```

Comment:
nitpick

```suggestion
        if not job_id:
```

Reply:
Again trying to be as explicit as above to ignore 0 or `''`. I do doubt the server would ever return these values though. 

`find_job_id_by_name` returns an int or None so the statement is easier to read and explicitly guarding against a None return statement.

I can change to if not job_id if needed though. Not opposed to making it `if not job_id`.

lmk what you think @Lee-W

Reply:
Sounds good 🙂

Reply:
resolved